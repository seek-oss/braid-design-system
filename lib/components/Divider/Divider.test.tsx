import React from 'react';
import { Divider } from '..';
import { render } from 'react-testing-library';
import forEachTheme from '../../test/utils/forEachTheme';

describe('Divider', () => {
  forEachTheme(({ ThemeProvider }) => {
    test('Default', () => {
      const { container } = render(
        <ThemeProvider>
          <Divider />
        </ThemeProvider>
      );
      expect(container.firstChild).toMatchSnapshot();
    });
  });
});
