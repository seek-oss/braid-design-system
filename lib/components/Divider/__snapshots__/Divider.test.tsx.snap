// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Divider Theme: jobStreet Default 1`] = `
<div
  class="Divider__root Box__root borderColor__standard"
>
  <div
    class="Divider__divider borderWidth__standard borderColor__standard"
  />
</div>
`;

exports[`Divider Theme: seekAnz Default 1`] = `
<div
  class="Divider__root Box__root borderColor__standard"
>
  <div
    class="Divider__divider borderWidth__standard borderColor__standard"
  />
</div>
`;

exports[`Divider Theme: seekAsia Default 1`] = `
<div
  class="Divider__root Box__root borderColor__standard"
>
  <div
    class="Divider__divider borderWidth__standard borderColor__standard"
  />
</div>
`;

exports[`Divider Theme: wireframe Default 1`] = `
<div
  class="Divider__root Box__root borderColor__standard"
>
  <div
    class="Divider__divider borderWidth__standard borderColor__standard"
  />
</div>
`;
